book/modules/appendix3.tex

```diff
@@ -48,16 +49,16 @@
 
 XXX Figure showing the little squares and triangles
 
-Equation \eqref{EQDETFORM} is beautiful and simple, but its derivation should give you pause. Volume Theorem I refers
+Equation \eqref{EQDETFORM} is beautiful and simple, but its derivation should give you pause. Volume theorem I refers
 to \emph{oriented} volume and we didn't make any reference to orientation in our figures! Indeed, we played tricks
-with pictures. We drew $\vec c_1$ and $\vec c_2$ in a right-handed orientation in the first quadrant, even though the
+with pictures. We drew $\vec c_1$ and $\vec c_2$ in a right-handed orientation\index{Basis!right-handed} in the first quadrant, even though the
 vectors $\mat{a\\c}$ and $\mat{b\\d}$ could be in \emph{any} quadrant (and one or both could even be the zero vector)!
 To fully justify Equation \eqref{EQDETFORM}, we need to consider cases based on all the possible ways $\vec c_1$ and $\vec c_2$
 can form a parallelogram. However, it turns out that every case gives the same answer: $\det\left(\mat{a&b\\c&d}\right)=ad-bc$.
 
 \begin{example}
-	Compute the determinant of $M=\mat{1 & 6 \\ 2 & 7}$ using both the $2\times 2$ formula and
-	the decomposition into elementary matrices.
+	Directly compute the determinant of $M=\mat{1 & 6 \\ 2 & 7}$ using the $2\times 2$ formula.
```

Comment:
Reworded because the original question could be interpreted as _use the formula and the elementary matrix decomposition together at the same time_.